### PR TITLE
Feature/sersors details

### PR DESCRIPTION
**Pull request by magnuscruz**
New features about sensors

### CODE REVIEW

**Comment by magnuscruz** [comment]
> LGTM 👍 The only thing missing is to update the database migration class to include the new schema and fields

What is DATABASE_VERSION value?

**Comment by hmatalonga** [comment]
https://github.com/greenhub-project/batteryhub/blob/b0956b4aeaa2d84c1aca2356b7c44f1cbedceec0/app/src/main/java/com/hmatalonga/greenhub/Config.java#L59

This is the current version, so in the migration you would include a new if block when coming from version 4 what to do. Finally you should bump this version number as well.

**Comment by magnuscruz** [comment]
I committed a new configuration. Please, can you review?

File: app/src/main/java/com/hmatalonga/greenhub/managers/storage/GreenHubDbMigration.java [hunk 1]
```diff
@@ -59,6 +59,37 @@ public void migrate(@NonNull DynamicRealm realm, long oldVersion, long newVersio
...
+            }
+
+            if (oldVersion == 4) {
+                objectSchema = schema.get("SensorDetails");
+                if (objectSchema != null) {
+                    schema.get("Sample")
+                            .addRealmListField("sensorDetailsList", objectSchema);
+                    oldVersion++;
+                }
+                /*objectSchema = schema.get("SensorDetails");
```
Comment (hmatalonga) [inline app/src/main/java/com/hmatalonga/greenhub/managers/storage/GreenHubDbMigration.java]: You also need to create the new object something along the lines of the example below:

```java
if (oldVersion == 4) {
        schema.create("SensorDetails")
            .addField("fifoMaxEventCount", int.class);
            ...
    oldVersion++;
}
```

Comment (magnuscruz) [reply app/src/main/java/com/hmatalonga/greenhub/managers/storage/GreenHubDbMigration.java]: Please, can you review?

**Pull request merged**